CHANGELOG.md

```diff
@@ -36,7 +85,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [0.20.0] - 2018-06-21
 ### Changed
-- Changed signature of `ESPProcess#initialize` to include a default value for `after_fork`. This prevents the 
+- Changed signature of `ESPProcess#initialize` to include a default value for `after_fork`. This prevents the
```

Comment:
Just sanity checking.. this does not break the API for existing code... because we are supplying default values ✅